docs/source/tutorials/basic_renewal_model.qmd

```diff
@@ -305,7 +332,7 @@ axes.plot(x_data, mean_Rt[0], color="C0", label="Mean")
 axes.legend()
 axes.set_title("Posterior Effective Reproduction Number", fontsize=10)
 axes.set_xlabel("Time", fontsize=10)
-axes.set_ylabel("$R_t$", fontsize=10)
+axes.set_ylabel("$\mathcal{R}(t)$", fontsize=10)
```

Comment:
There is a warning on the website:
```pycon
<>:29: SyntaxWarning: invalid escape sequence '\m'
<>:29: SyntaxWarning: invalid escape sequence '\m'
/tmp/ipykernel_2398/2465682994.py:29: SyntaxWarning: invalid escape sequence '\m'
  axes.set_ylabel("$\mathcal{R}(t)$", fontsize=10)
```

Reply:
good catch, needs to be a double backslash to escape it.